2021_CommandBased/src/main/java/frc/robot/util/KalmanFilterLatencyCompensator.java

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
How was this modified from the default version?

Reply:
the constructor was made public

Reply:
Perhaps extending the included version of KalmanFilterLatencyCompensator and giving it a new name would be a better way to achieve this?

Reply:
I can't extend the wpilib version because it doesn't have a public constructor